java/overrides.js

```diff
@@ -38,6 +38,9 @@
     'check regions by coordinates in overflowed frame with css stitching': {skip: true}, // com.applitools.eyes.OutOfBoundsException: Region [(0, 15000) 385x5000, SCREENSHOT_AS_IS] is out of screenshot bounds [(0, 0) 385x15000, SCREENSHOT_AS_IS]
     'check regions by coordinates in overflowed frame with scroll stitching': {skip: true}, // com.applitools.eyes.OutOfBoundsException: Region [(0, 15000) 385x5000, SCREENSHOT_AS_IS] is out of screenshot bounds [(0, 0) 385x15000, SCREENSHOT_AS_IS]
     'check regions by coordinates in overflowed frame with vg': {skip: true}, // Unable to locate element: {"method":"css selector","selector":"#modal3 iframe"}
+    'check scrollable modal region by selector fully with css stitching': {config: {branchName: 'current1'}}, // diffs if compare to common baseline
+    'check scrollable modal region by selector fully with scroll stitching': {config: {branchName: 'current1'}}, // diffs if compare to common baseline
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
    "check scrollable modal region by selector fully with scroll stitching": {config: {branchName: 'current1'}}, // diffs if compare to common baseline
```